examples/image_classification-tf.ipynb

```diff
@@ -0,0 +1,7883 @@
+{
```

Comment:
<p>It's probably worth mentioning that the samples of <code>train_ds</code> and <code>val_ds</code>  have two keys - <code>pixel_values</code> and <code>label</code></p><p> </p><p>These <code>label</code>are used by HF to compute the model's loss internally.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>